src/uu/test/src/parser.rs

```diff
@@ -121,6 +121,9 @@ impl Parser {
     /// Test if the next token in the stream is a BOOLOP (-a or -o), without
     /// removing the token from the stream.
     fn peek_is_boolop(&mut self) -> bool {
+        // matches!(self.peek(), Symbol::BoolOp(_))
+        #[allow(clippy::match_like_matches_macro)]
```

Comment:
attributes are not yet allowed on `if` expressions :)

Reply:
Hmm, to use attributes on 'if' expressions Rust 1.43 is needed... *sigh* I give up.